### PR TITLE
Chore: Extracts priceFormatter hook

### PR DESCRIPTION
**Pull request by hellofanny**
## What's the purpose of this pull request?

We have been using priceFormatter in many components examples. So I extracted it to the `Helpers` folder.

## How to test it?

Everything should work as before.

### CODE REVIEW

**Review by renatamottam — APPROVED** [review]
Nice one!

File: apps/site/pages/components/molecules/gift.mdx [hunk 1]
```diff
@@ -142,8 +132,8 @@ export const GiftTemplateWithoutBadge = (
 
 <Tabs items={['Example', 'Code']} defaultIndex="0">
   <Tab>
-    <OverviewSection>{GiftTemplate} {GiftTemplateIcon}</OverviewSection>
-    <OverviewSection>{GiftTemplateWithoutIcon} {GiftTemplateWithoutBadge}</OverviewSection>
+    <OverviewSection bigGap>{GiftTemplate} {GiftTemplateIcon}</OverviewSection>
```
Comment (eduardoformiga) [inline apps/site/pages/components/molecules/gift.mdx:135]: 👀  nice!

**Review by eduardoformiga — APPROVED** [review]
LGTM! the capitalized folder is intentional, right?

**Comment by hellofanny** [comment]
> LGTM! the capitalized folder is intentional, right?

@eduardoformiga it was, but I don't know if should hahah. WDYT?

**Comment by hellofanny** [comment]
> > LGTM! the capitalized folder is intentional, right?
> 
> @eduardoformiga it was, but I don't know if should hahah. WDYT?

I've renamed it to `uitlities` and left it lowercase.

**Pull request merged**